tests/openapi/test_strictmode.py

```diff
@@ -1010,7 +1013,10 @@ def test_strict_mode_write_rate_limiting(collection_name):
         if not response.ok:
             failed_count += 1
             assert response.status_code == 429
-            assert "Rate limiting exceeded: Write rate limit exceeded, retry later" in response.json()['status']['error']
+            assert "Rate limiting exceeded: Write rate limit exceeded" in response.json()['status']['error']
+            assert response.headers['Retry-After'] is not None
+            # need to wait 59s for the single token available to be replenished
+            assert int(response.headers['Retry-After']) is 59
```

Comment:
I vote to relax the requirement of 59 a bit. Maybe we can pick the range [50,60]? 60 because of rounding up as described below.

```suggestion
            # need to wait 59s for the single token available to be replenished
            assert int(response.headers['Retry-After']) is 59
```

Maybe we want another rate limit test that asserts we get a low retry-after value if:
- our read limit is high
- we've exhausted reads
- we'll have enough tokens for the next read almost immediately

Wdyt?

Reply:
Implemented in a range from 55 to 60.